src/web/lib/hakuneko/frontend@classic-dark/menu.html

```diff
@@ -142,6 +147,13 @@
                 <div class="logo">
                     <img src="./img/logo_m.png"/>
                 </div>
+                <div class="contact">
+                    <div >
+                        <img src="http://icons.iconarchive.com/icons/papirus-team/papirus-apps/48/discord-icon.png"/>
+                       
+                            <a mat-raised-button href="https://discord.gg/A5d3NDf" color="primary">Contact</a>
```

Comment:
Seems there are some material design directives used for this anchor element such as `mat-raised-button` attribute and `primary` class, these are not available in HakuNeko